### DISCUSSION TITLE
Npm run dev issue with vite

### DISCUSSION THREAD

**Discussion by Michvista** [sveltejs/kit#10097] [Q&A]
Everytime i run: npm run dev i get this error: Failed to scan for dependencies from entries:
  C:\Users\dell\Desktop\all\enhanzer\node_modules\@sveltejs\kit\src\runtime\client\start.js
My svelte kit version is 1.20.1 but previously it was 1.5.0 before i updated it to 1.20.1
please help🙏🏾

**Comment by teemingc** [comment]
Hi Michvista, can you provide a sample repository where this error occurs? Also, what version of node.js are you using? Have all the dependencies been installed correctly?

**Reply by Michvista** [reply]
Im using nodejs 18+ amd yes all the dependencies have been installed correctly as for the error pls give me a few hours to get the sample

**Comment by minvernizzinextbit** [comment]
@eltigerchino @Michvista  Today this happened to me as well but what I found out was that the error was actually caused by a VSCode extension that I updated and was now missing a folder, thus breaking the Vite dev server.
In my case the extension was [Sherlock by inlang](https://inlang.com/m/r7kp499g/app-inlang-ideExtension) and I noticed it because VSCode was throwing an error on startup. Adding the missing folder, restarting VSCode and of course restarting the Vite dev server resolved the issue for me.
I hope this helps!

**Reply by dominikg** [reply]
> was now missing a folder, thus breaking the Vite dev server. In my case the extension was [Sherlock by inlang](https://inlang.com/m/r7kp499g/app-inlang-ideExtension) 

which folder was missing and should that extension create it itself? (or are you talking about the generated modules by paraglide?) cc @LorisSigrist

**Reply by minvernizzinextbit** [reply]
Hi @dominikg, I'm talking about /_[project-name]_.inlang/cache/modules. I can see why the Sherlock extention cannot create it by itself (as the name and location of the inlang project could vary depending on repo structure).

**Reply by minvernizzinextbit** [reply]
I also should mention that while debugging I tried many different things (reinstalled node_modules, trying different versions of pnpm since I had just updated it, etc.), so the extention might not be the root cause of the error. If I have time I will try to reproduce the error more diligently.

**Reply by LorisSigrist** [reply]
Hi @minvernizzinextbit, we're already working on a fix. 
In the meantime creating a `cache/modules` folder inside your `project.inlang` directory should fix the issue

Edit: Update is being released, please update your `@inlang/paraglide-sveltekit` version
